clientreceipt/clientreceipt.sol

```diff
@@ -68,16 +69,16 @@ contract ClientReceipt is owned, killswitch {
   event Transfer(address indexed _from, address indexed _to, uint _value);
   
   function() {
-	AnonymousDeposit(msg.sender, msg.value)
+	AnonymousDeposit(msg.sender, msg.value);
   }
   function deposit(hash _id) {
-	Deposit(msg.sender, _id, msg.value)
+	Deposit(msg.sender, _id, msg.value);
   }
   function refill() {
-	Refill(msg.sender, msg.value)
+	Refill(msg.sender, msg.value);
   }
-  function transfer(address _to, uint _value, byte[] _data) restricted {
+  function transfer(address _to, uint _value, bytes _data) restricted {
 	Transfer(msg.sender, _to, _value);
-	_to.send(_value, _data);
+	_to.value(_value).call(_data);
```

Comment:
Sorry, I think I told you to use this. It should actually be `_to.call.value(_value)(_data);`.